ui/Screen/Help.svelte

```diff
@@ -1,40 +1,58 @@
 <script>
   import ModalLayout from "../DesignSystem/Component/ModalLayout.svelte";
+
+  const OSKey = navigator.platform.includes("Mac") ? "⌘" : "ctrl";
```

Comment:
Actually this should be `modifierKey`.

Reply:
What do you mean?

Reply:
I have both a mac and windows at home and all the actions I do with the `cmd` key are mapped to `ctrl` on windows.

Reply:
I meant the name of the variable.

Reply:
So `OSKey` -> `modifierKey`.

Reply:
Oh you mean the variable?

Reply:
We do the same check in `Hotkeys.svelte`, maybe worth moving to lib code.

Reply:
+1 on naming it `modifierKey` in both cases.

Reply:
https://github.com/radicle-dev/radicle-upstream/commit/79c719e7030813f8769a00892116d4b1c31397b3